README.md

```diff
@@ -1,150 +1,119 @@
 # workshop-template
 
 This repository is [Software Carpentry](http://software-carpentry.org)'s
-template for creating websites for workshops.  Do *not* fork this
-repository directly on GitHub.  Instead, follow the instructions below
-to create a website repository for your workshop (and possibly a
-second repository for your learners to use in your Git lessons).
-
-> If you are teaching Git, you should create a separate repository for
-> learners to use in that lesson.  You should not have them use the
-> workshop website repository because:
+template for creating websites for workshops.
+Do *not* fork this repository directly on GitHub.
```

Comment:
Line 5 - Relocate the warning on forking to the **Creating a Repository** section.
Line 6 - Remove the word 'Instead,'

Although the warning is important, it seems misplaced now that the **Creating a Repository** process has been improved.